RSSDP/RSSDP.csproj

```diff
@@ -14,6 +14,7 @@
   <PropertyGroup>
     <TargetFramework>netstandard2.1</TargetFramework>
     <GenerateAssemblyInfo>false</GenerateAssemblyInfo>
+    <TreatWarningsAsErrors>true</TreatWarningsAsErrors>
```

Comment:
Is there a reason this does not have the same condition and the one in Jellyfin.Data?

Reply:
Yes, Jellyfin.Data has analyzers which run on debug builds and those still spit out warnings